src/main/java/com/mercadolibre/grupo1/projetointegrador/entities/PurchaseItem.java

```diff
@@ -1,4 +1,32 @@
 package com.mercadolibre.grupo1.projetointegrador.entities;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+/**
+ * classe responsavel por registrar os informacoes de um item dentro do compra do cliente
+ * @Author: Rogerio Lambert
+ */
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Vamos seguir esse padrao de deixar as tabelas no plural
```suggestion
@NoArgsConstructor
@Table(name = "purchase_items")
```

Reply:
Faz sentido padronizarmos os nomes das tabelas mesmo
Vou acrescentar